enterprise-suite/values.yaml

```diff
@@ -2,7 +2,7 @@
 # Update these versions for a release.
 #
 # lightbend-docker-registry.bintray.io/enterprise-suite/es-console
```

Comment:
We should change these comments now too where appropriate.  (Add the `commercial` bit.)